Makefile

```diff
@@ -3,13 +3,13 @@
 
 check_dirs := examples tests src utils
 
-# get modified files since the branch was made
-fork_point_sha := $(shell git merge-base --fork-point master)
-joined_dirs := $(shell echo $(check_dirs) | tr " " "|")
-modified_py_files := $(shell git diff --name-only $(fork_point_sha) | egrep '^($(joined_dirs))' | egrep '\.py$$')
-#$(info modified files are: $(modified_py_files))
 
 modified_only_fixup:
+	# get modified files since the branch was made
```

Comment:
This part has nothing to do with this PR, it should go in a separate PR. Make sure to tag stas00 since he is the one who updated those changes in the Makefile and might have some ideas on how to best fix it to work on Windows.